vault/models/vault_entry.py

```diff
@@ -1,17 +1,21 @@
 # © 2021 Florian Kantelberg - initOS GmbH
+# Copyright 2022 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 import logging
 from datetime import datetime
 from uuid import uuid4
 
-from odoo import _, api, fields, models
+import odoo
+from odoo import _, api, fields
 from odoo.exceptions import ValidationError
 
+from odoo.addons.web.models import models
+
 _logger = logging.getLogger(__name__)
 
 
-class VaultEntry(models.Model):
+class VaultEntry(odoo.models.Model):
```

Comment:
I insist: why this change?

Reply:
It's because he is adding this models: https://github.com/OCA/server-auth/pull/411/files#diff-40d91c6c9961012399719d24970edab4a33704ffb9d72e74ba0528c9b966f378R13

Reply:
Then better to do `from odoo.addons.web.models import models as web_models`

Reply:
Finally, this is not necessary.